core/trino-main/src/test/java/io/trino/sql/planner/TestUnwrapCastInComparison.java

```diff
@@ -815,20 +818,32 @@ private void testRemoveFilter(String inputType, String inputPredicate)
         }
     }
 
+    private void testUnwrap(String inputType, String inputPredicate)
```

Comment:
This isn't quite a normal overload of the `testUnwrap(String inputType, String inputPredicate, String expectedPredicate)` method

what about calling this differently?
like `testEliminateFalseFilter`?